vagrant-newdisk.gemspec

```diff
@@ -6,12 +6,12 @@ require 'vagrant/newdisk/version'
 Gem::Specification.new do |spec|
   spec.name          = "vagrant-newdisk"
   spec.version       = Vagrant::Newdisk::VERSION
-  spec.authors       = ["Yavor Nikolov"]
-  spec.email         = ["nikolov.javor@gmail.com"]
+  spec.authors       = ["Yavor Nikolov", "Jérôme Martin"]
+  spec.email         = ["nikolov.javor@gmail.com", "jerome.f.martin@gmail.com"]
 
-  spec.summary       = %q{Vagrant plugin to add VirtualBox disk}
-  spec.description   = %q{Vagrant plugin to add VirtualBox disk at creation time}
-  spec.homepage      = "https://github.com/dbfit/vagrant-newdisk"
+  spec.summary       = %q{Vagrant plugin to add VirtualBox/Hyper-V disk}
+  spec.description   = %q{Vagrant plugin to add VirtualBox/Hyper-V disk at creation time}
+  spec.homepage      = "https://github.com/jeromemartin/vagrant-newdisk"
```

Comment:
Should we change that `homepage` url considering that we're merging these changes under `dbfit/vagrant-newdisk`?

Reply:
If the PR is accepted, it should not be changed